spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -401,6 +401,20 @@ public void validate(InjectionPoint injectionPoint);
 
 ==== Firing an event
 
+The method `BeanManager.getEvent()` returns an an instance of `Event<Object>` and without qualifier.
```

Comment:
Given that this method should simulate `@Inject Event<Object>` I think we should use something like:

The method `BeanManager.getEvent()` returns an instance of `Event` with _specified type_ `java.lang.Object` and _specified qualifier_ `@Default`.

See also 3.8. "The default qualifier at injection points". Terms specified type and specified qualifiers are used in "10.2.3. The Event interface".

Hm, but it might be better to use the  _specified type_ `java.lang.Object` and _specified qualifier_ `@Any`. WDYT?